README.md

```diff
@@ -1,26 +1,197 @@
 # Rails Layouts And Templates
 
+This lesson will show you how to use layouts to achieve a common look and feel between different views in your app.
+
 ## Objectives
 
+After this lesson, you'll be able to...
+
 1. Identify the default application layout.
 2. Yield to view templates from a layout.
 3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option.
 4. Use :layout => false to shut off the layout and only render the view
 
-## Outline
+## Life Without Layouts
+
+Imagine that you're tasked to build an online store app with Rails.
+
+You would probably have a few different views in this app, for example:
+
+1. A list of products
+2. A detail view that shows more info for a selected product
+3. A shopping cart
+
+Across all these views you would want a consistent look, which is called a layout. This layout perhaps contains something like a logo, navigation links, a search bar, and a footer at the bottom that contains some info about the shop.
+
```

Comment:
Good slow easing into layouts. You're using concrete, real-world concepts here to draw in